src/endpoints.ts

```diff
@@ -0,0 +1,25 @@
+import * as https from 'https';
```

Comment:
I don't think this will be included, at least not in our frontend builds that only care about the participant client. @ProbablePrime can you check?